ganymede/models/stahl/cd_locate_stores.sql

```diff
@@ -0,0 +1,9 @@
+{{ config(materialized='view') }}
```

Comment:
let's name models as objects as opposed to phrases, functions... you can call this `dim_stores` if you want. It's popular naming convention. read more here - https://stackoverflow.com/questions/1716220/star-schema-naming-conventions

Reply:
Let's use this as a dim table for stores where we denormalize all dimensions of the store and anticipate its use case in downstream wolfram resources. I.e. let's flatten and expand all the columns and alias the names.